tensorflow/core/kernels/diag_op_gpu.cu.cc

```diff
@@ -62,16 +62,23 @@ struct DiagFunctor<GPUDevice, T> {
     const GPUDevice& device = context->eigen_device<GPUDevice>();
     GpuLaunchConfig diag_config =
         GetGpuLaunchConfig(virtual_thread_count, device);
-    TF_CHECK_OK(
-        CudaLaunchKernel(DiagCudaKernel<T>, diag_config.block_count,
-                         diag_config.thread_per_block, 0, device.stream(),
-                         diag_config.virtual_thread_count, size, in, out));
+    TF_CHECK_OK(GpuLaunchKernel(
+        DiagGpuKernel<T>, diag_config.block_count, diag_config.thread_per_block,
+        0, device.stream(), diag_config.virtual_thread_count, size, in, out));
 
+#if GOOGLE_CUDA
```

Comment:
This can be removed, the error is already checked in GpuLaunchKernel. Thanks.

Reply:
@chsigg thanks for reminding, will do.

@whchung This review feedback make me realize the `hipGetLastError()` helper function. Before `hipLaunchKernelGGLEx()` is approved, should we be able to use the `hipGetLastError()` as below to process the error code inside [`GpuLaunchKernel()`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/util/gpu_kernel_helper.h#L80)